test-fixture/entity/src/main/kotlin/com/linecorp/kotlinjdsl/test/entity/EntityDsl.kt

```diff
@@ -12,6 +12,11 @@ import javax.persistence.EntityManager
 abstract class EntityDsl {
     fun EntityManager.persistAll(vararg entities: Any) = persistAll(entities.toList())
     fun EntityManager.persistAll(entities: Collection<Any>) = entities.forEach { persist(it) }
+    fun EntityManager.persistAllFlushAndClearEach(vararg entities: Any) = persistAllFlushAndClearEach(entities.toList())
```

Comment:
-- korean
이 코드를 추가한 이유는 eclipselink 가 persist 를 list 로 처리한후 한번에 flush 를 할 경우 persist 호출 순서대로 엔티티의 디비 삽입을 보장하지 않았기 때문입니다.

-- english
The reason for adding this code is that if eclipselink processes persist as a list and then flushes it at once, it does not guarantee inserting the entity into DB in the order in which persist is called.

Reply:
-- korean
옛날 글이지만 eclipselink 입장에서는 JPA 스펙에 저장 순서에 대한 내용이 없기 떄문에 보장하지 않는다고 하는군요.

-- english
It's an old thread, but they don't guarantee it because there's no spec on the order of persist in the JPA .

https://www.eclipse.org/lists/eclipselink-users/msg07683.html
> Not to turn this into a Hibernate versus Eclipselink thread, but given that the JPA spec does not
specify the ordering behavior you are seeking (since it apparently is not relevant to the core
JPA requirments) you have NO guarantee that Hibernate will perform as you wish in future
versions. You don't even know if it will perform as you wish against a different database vendor or
if the wind is blowing from the north instead of the south on a given day, etc.

Reply:
-- korean
그렇다면 저장 순서를 보장해서 테스트를 통과 시키는 것이 아니라 테스트의 verify 부분을 `orders.minByOrNull { it.id }` 로 변경하는 것이 좋을 것 같습니다.

왜냐하면 `AbstractTest`들은 JPA 구현체와 상관 없이 JPA 스펙으로만 작성 되어야 하기 때문입니다.

-- english
If so, it would be better to change the verification part of the test to `order.minByOrNull { it.id }` rather than to ensure the order of storage.

This is because the `AbstractTest` should be written only in the JPA specification regardless of the JPA implementation.

Reply:
-- korean
추가로 persist를 할 때마다 flush and clear를 하는 것은 테스트 성능에도 좋지 않을 것으로 보입니다.

-- english
Additionally, flush and clear in every persist will not be good for test performance.

Reply:
-- korean
리뷰 감사합니다. 테스트 코드를 JPA 스펙에 맞게 삽입 순서와 관련 없이도 통과하도록 수정하였습니다. 그리고 말씀해주신 코드는 삭제했습니다.(persistAllFlushAndClearEach)

-- english
Thanks for the review. The test code has been modified to pass regardless of the insertion order according to the JPA specification. and your review applied (delete persistAllFlushAndClearEach)